docs/03_Get_Started/01_Get_started_with_ESP32_and_Arduino.md

```diff
@@ -0,0 +1,64 @@
+
+In this guide you will learn how to play with the **Arduino** framework and the *ESP32* microcontroller of your new **UDOO KEY board**.  
+It is quite easy since it works like any other ESP32 based board.
+
+First of all, let's download and install the Arduino IDE following the [official documentation](https://support.arduino.cc/hc/en-us/articles/360019833020-Download-and-install-Arduino-IDE).
+
+Now you have to properly setup the IDE in order to use it with the board.
+
+Leave opened the jumper labeled with `Serial Sel` to drive the USB communication to the ESP32 and connect the board to your PC using a USB cable.
```

Comment:
`s/opened the jumper labeled with `Serial Sel`/the jumper labeled with `Serial Sel` open/`